config.go

```diff
@@ -17,19 +17,20 @@ import (
 )
 
 const (
-	optionPrefix      = "agent."
-	logLevelFlag      = optionPrefix + "log"
-	logsVSockPortFlag = optionPrefix + "log_vport"
-	devModeFlag       = optionPrefix + "devmode"
-	traceModeFlag     = optionPrefix + "trace"
-	useVsockFlag      = optionPrefix + "use_vsock"
-	debugConsoleFlag  = optionPrefix + "debug_console"
-	kernelCmdlineFile = "/proc/cmdline"
-	traceModeStatic   = "static"
-	traceModeDynamic  = "dynamic"
-	traceTypeIsolated = "isolated"
-	traceTypeCollated = "collated"
-	defaultTraceType  = traceTypeIsolated
+	optionPrefix          = "agent."
+	logLevelFlag          = optionPrefix + "log"
+	logsVSockPortFlag     = optionPrefix + "log_vport"
+	devModeFlag           = optionPrefix + "devmode"
+	traceModeFlag         = optionPrefix + "trace"
+	useVsockFlag          = optionPrefix + "use_vsock"
+	debugConsoleFlag      = optionPrefix + "debug_console"
+	debugConsoleVPortFlag = optionPrefix + "debug_console_vport"
```

Comment:
Could you update `config_test.go` for this new value. Also, can you add it to the `README.md` please.